include/xtypes/AliasType.hpp

```diff
@@ -113,6 +113,31 @@ class AliasType : public DynamicType
         return *aliased_;
     }
 
+    const DynamicType& get()
+    {
+        return *aliased_;
+    }
+
+    const DynamicType* operator -> () const
+    {
+        return aliased_.get();
+    }
+
+    const DynamicType* operator -> ()
+    {
+        return aliased_.get();
+    }
+
+    const DynamicType& operator * () const
+    {
+        return *aliased_;
+    }
+
+    const DynamicType& operator * ()
```

Comment:
The returned type should be non-const

Reply:
These methods were finally removed as they could result in a security vulnerability.